src/plugins/auto-moderation-plugin.ts

```diff
@@ -0,0 +1,161 @@
+import fs from 'node:fs'
+
+import type { PluginContext, PluginInterface } from '../common/plugins.js'
+
+import { EventType, InstanceType, PunishmentType, Severity, ChannelType } from 'src/common/application-event.js'
+import { antiSpamString } from 'src/util/shared-util.js'
```

Comment:
You need to use the relative path
```suggestion
import { EventType, InstanceType, PunishmentType, Severity, ChannelType } from '../common/application-event.js'
import { antiSpamString } from '../util/shared-util.js'
```